### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andreyferriyan**
Update README.md in installation section. To avoid error with different shell.

### CODE REVIEW

**Comment by andreyferriyan** [comment]
> It looks like @andreyferriyan hasn't signed our **C**ontributor **L**icense **A**greement, yet.
> 
> > The purpose of a CLA is to ensure that the guardian of a project's outputs has the necessary ownership or grants of rights over all contributions to allow them to distribute under the chosen licence.
> > [Wikipedia](http://en.wikipedia.org/wiki/Contributor_License_Agreement)
> 
> You can read and sign our full Contributor License Agreement [here](http://clabot.confluent.io/cla).
> 
> Once you've signed reply with `[clabot:check]` to prove it.
> 
> Appreciation of efforts,
> 
> clabot

`[clabot:check]`

File: README.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ Install
           its dependencies using the repositories below and then build
           confluent-kafka  using the command in the "Install from
           source from PyPi" section below.
-
+
```
Comment (edenhill) [inline README.md]: remove whitespace diff

File: README.md [hunk 2]
```diff
@@ -203,7 +203,11 @@ Install
     $ pip install --no-binary :all: confluent-kafka
 
 
-For source install, see *Prerequisites* below.
+For source install, see *Prerequisites* below.   
+
+To avoid error in installation like *no matches found: confluent-kafka[avro]*, use double quotes. As an example:
+
+    $ pip install "confluent-kafka[avro]"
```
Comment (edenhill) [inline README.md]: I think it is better to just change the "pip install ..avro]" line above to include quotes, people tend to cutnpaste the first command they see

Comment (andreyferriyan) [reply README.md]: Sorry, this is my first time to change. Where should I change the line ?. In my forking repo ?

Comment (edenhill) [reply README.md]: Yes, undo your previous changes and modify the original `pip install confluent-kafka[avro]` line above to have quotes. Commit and push.

File: README.md (hunk 1 above)
Comment (edenhill) [reply README.md]: remove the whitespace diff

File: README.md [hunk 3]
```diff
@@ -189,21 +189,21 @@ Install
...
-    $ pip install confluent-kafka[avro]
+    $ pip install "confluent-kafka[avro]"
 
 **Install from source from PyPi** *(requires librdkafka + dependencies to be installed separately)*:
 
     $ pip install --no-binary :all: confluent-kafka
 
 
-For source install, see *Prerequisites* below.
+For source install, see *Prerequisites* below.
```
Comment (edenhill) [inline README.md]: remove the whitespace diff

Comment (edenhill) [reply README.md]: This whitespace change needs to be removed too.

**Pull request merged**